README.md

```diff
@@ -138,7 +138,7 @@ superpowers:
 
 ### git am, git apply
 
-    $ git am https://github.com/defunkt/hub/pull/55
```

Comment:
Sorry, I don't want to change these examples. These don't have to be real URLs, and having "github" twice in the URL might be confusing, even though that's the latest name of this repo. I might change the name of the example repo in the future.